app/views/components/pageflow/admin/entries_tab.rb

```diff
@@ -3,17 +3,20 @@ module Admin
     class EntriesTab < ViewComponent
       def build(theming)
         account = theming.account
-        if account.entries.any?
-          table_for account.entries, :i18n => Pageflow::Entry do
-            column :title do |entry|
+        embedded_index_table account.entries.order(
+            params[:order] && Entry.column_names.include?(params[:order].gsub('_asc', '').gsub('_desc', '')) ?
```

Comment:
Line is too long. [110/100]<br>Indent the first parameter one step more than the start of the previous line.<br>Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead.